lib/p2p/Peer.ts

```diff
@@ -17,6 +17,7 @@ import addressUtils from '../utils/addressUtils';
 import NodeKey from '../nodekey/NodeKey';
 import Network from './Network';
 import Framer from './Framer';
+import { cat } from 'shelljs';
```

Comment:
Looks like this is not being used.

Reply:
Thanks, I think this was a typo where I accidentally hit enter and this import was automatically created.